ui-tests/cypress/support/pageObjects/content/types/TypesPage.js

```diff
@@ -1,5 +1,69 @@
-import {TEST_ID_KEY, htmlElements, WebElement} from "../../WebElement.js";
+import {TEST_ID_KEY, htmlElements, WebElement} from '../../WebElement.js';
+import Content from '../../app/Content.js';
+import AppPage from '../../app/AppPage.js';
+import AddPage from './AddPage.js';
+import EditPage from './EditPage.js';
 
-import Content from "../../app/Content.js";
+export default class TypesPage extends Content {
+  
+  editOption = `${htmlElements.li}.ContentTypeList__menu-item-edit`;
+  deleteOption= `${htmlElements.li}.ContentTypeList__menu-item-delete`;
+  modalDeleteButton = `${htmlElements.button}#DeleteContentTypeModal__button-delete`;
 
-export default class TypesPage extends Content {}
\ No newline at end of file
+  getAddButton() {
+    return this.getContents()
+               .find(htmlElements.button)
+               .contains('Add');
+  }
+
+  getTable() {
+    return this.getContents()
+               .find(htmlElements.table);
+  }
+
+  getTableCell(content) {
```

Comment:
Retrieving elements needed for validation should be responsibility of the test script not the page object